ceph.spec.in

```diff
@@ -174,7 +174,9 @@ BuildRequires:	libnl3-devel
 BuildRequires:	liboath-devel
 BuildRequires:	libtool
 BuildRequires:	libxml2-devel
+%if 0%{with make_check}
 BuildRequires:	libuuid-devel
+%endif
```

Comment:
instead of just blocking it in a conditional, would it make more sense to move the line `BuildRequires: libuuid-devel` into the main `%if 0%{with make_check}` block, below?

Reply:
sure. fixed and repushed.